client/src/components/Auth/SignUpScreen.tsx

```diff
@@ -26,14 +28,23 @@ const SignUpScreen = () => {
   const [email, setEmail] = React.useState<string>("");
   const [password, setPassword] = React.useState<string>("");
   const [confirmPassword, setConfirmPassword] = React.useState<string>("");
-
+  const [errorVisible, setErrorVisible] = React.useState<('none' | 'flex' | undefined)>("none");
+  const [errorText, setErrorText] = React.useState<string>("")
+  const clearForm = () => {
+      setEmail("");
```

Comment:
I would say that we should not clear there entire form, maybe just the passwords as they may still want to use that email. Less of a headache to users.